students/etwum/lesson05/except_exercise.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python
+
+"""
+An exercise in playing with Exceptions.
+Make lots of try/except blocks for fun and profit.
+
+Make sure to catch specifically the error you find, rather than all errors.
+"""
+
+from except_test import fun, more_fun, last_fun
+
+
+# Figure out what the exception is, catch it and while still
+# in that catch block, try again with the second item in the list
+first_try = ['spam', 'cheese', 'mr death']
+try:
+    joke = fun(first_try[1])
+except NameError as e:
+    print(e)
```

Comment:
You want to 'try' with the first item in the list here `first_try[0]` and then with the second `first_try[1]` in the `except` block